docs/blog/2024-09-12-next-gen-module-federation-deployments.md

```diff
@@ -0,0 +1,549 @@
+---
+title: Next-Gen Module Federation Deployments with Nx and Zephyr Cloud
+slug: next-gen-module-federation-deployment
+authors: ['Colum Ferry']
+tags: [nx, module federation, rspack, zephyr cloud]
+cover_image: /blog/images/2024-09-10/personal-access-tokens-header.avif
+published: false
+---
+
+Nx has supported Module Federation for a long time now and its popularity is seeing year-on-year increase. It has proven
+to be the defacto solution for modern Micro Frontends and Nx has embraced this and helped developers scaffold projects
+that use Module Federation; offering the most scalable and the most developer friendly experience to do so.
+However, we have always taken a step back when it came to the final piece of the puzzle. Deployment.
+Micro Frontends are, by their very nature, more complex to deploy. You are no longer deploying one build artifact but
+rather, many build artifacts that could even be on different release cadences from each other.
+And yet, we still expect the final application that our users interact with to be stable.
+
+There are so many approaches to deployments - think Kubernetes, Storage Buckets with DNS, EC2 instances etc - that it
+has always been difficult for us to recommend a solution.
+
+That was until [Zephyr Cloud](https://www.zephyr-cloud.io/) entered the game.
+
+Zephyr Cloud is a "bring your own cloud" deployment platform with best-in-class support for module federation projects
+that integrates right into your application build, providing seamless deployments without having to learn any new
+commands.
+With features such as:
+
+- Auto-deployment on build
+- Live Preview link generation
+- Version rollback and roll-forward
+- Micro Frontend dependency management
+- A Chrome Extension to manage environments
+- Support for a wide range of Cloud Platforms (such as [Cloudflare](https://www.cloudflare.com/en-gb/)
+  and [Netlify](https://www.netlify.com/))
+- A beautiful UI to visualize your deployments
+
+And so much more, Zephyr Cloud really is one of the best deployment solutions available.
+
+{% callout type="note" title="Zephyr Cloud Features" %}
+You can learn more about the full feature set that Zephyr Cloud
+offers [here](https://docs.zephyr-cloud.io/#main-features).
+{% /callout %}
+
+I myself, Colum Ferry, after having been allowed to experiment with it in its infancy back in February 2024 said
+
+> Zephyr Cloud is the Micro Frontend orchestration tool you never knew you needed.
+> What they have built will change the deployments for the better.
+> Think of the disruption k8s caused. Zephyr Cloud will do the same for the Micro Frontend world.
+
+<small>
+Linkedin: https://www.linkedin.com/posts/colum-ferry-3a36a9169_zephyr-cloud-is-the-micro-frontend-orchestration-activity-7183207192991289344-6mSR</small>
+
+And I stand by that statement.
```

Comment:
the `<small>` don't seem to render here
![image](https://github.com/user-attachments/assets/364924df-1b5e-4b49-a57c-218bab4bf1f3)